build-bin/docker/configure_docker_push

```diff
@@ -50,5 +50,8 @@ fi
 # See https://github.com/multiarch/qemu-user-static
 #
 # Mirrored image use to avoid docker.io pulls:
-# docker tag multiarch/qemu-user-static:5.1.0-7 ghcr.io/openzipkin/multiarch-qemu-user-static:latest
+# docker tag multiarch/qemu-user-static:7.2.0-1 ghcr.io/openzipkin/multiarch-qemu-user-static:latest
```

Comment:
@anuraaga fyi I will re-cut 3.8.5 after this, to make sure the much newer version of qemu still works. 3.8.5 isn't in any prod image, yet.

Reply:
cool nothing blew up https://github.com/openzipkin/docker-alpine/actions/runs/7094978815/job/19311172296